### ISSUE TITLE
No Warning or Message if you create a Queue with an already-used URL

### ISSUE THREAD

**Issue by williamg** [jez/cmd-queue#1]
If you create a queue with a unique name, but manually set the address to one that is already used, it acts as if your queue was created, but it isn't visible. Perhaps an error message should be displayed.

**Comment by jez** [comment]
Yeah, I need to add error handling in most places. I'm pretty sure it'll never let you do anything that shouldn't be possible, but it won't tell the user when that happens sometimes.
